src/sagemaker/tensorflow/README.rst

```diff
@@ -15,485 +34,130 @@ is a two-step process.
 First, you prepare your training script, then second, you run it on
```

Comment:
ll. 32-33: maybe change to "a `sagemaker.tensorflow.TensorFlow` estimator"? or remove the first "a"